src/state/pipeline-run-task-steps/index.js

```diff
@@ -0,0 +1,7 @@
+import { makeLocalGetter } from '../../utils/object';
+import get from 'lodash/get';
+
+const localGetter = makeLocalGetter('pipelineRunTaskSteps');
+
+export const getPipelineRunTaskSteps = (state) =>
+  get(state, 'pipelineRunTaskSteps', []);
```

Comment:
remove unused getter
```suggestion
import get from 'lodash/get';

export const getPipelineRunTaskSteps = (state) =>
  get(state, 'pipelineRunTaskSteps', []);
```